cmd/system-probe/modules/language_detection.go

```diff
@@ -66,7 +70,7 @@ func handleError(writer http.ResponseWriter, status int, err error) {
 	writer.WriteHeader(status)
 }
 
-func detectLanguage(writer http.ResponseWriter, request *http.Request) {
+func (l languageDetectionModule) detectLanguage(writer http.ResponseWriter, request *http.Request) {
```

Comment:
I think you want pointer receivers here too

Reply:
Ah thank you, I changed it for the `PrivilegedLanguageDetector` but forgot to change it for the `languageDetectionModule`. Sorry for making you check twice 🙇🏻‍♀️